audio/libsndfile/Portfile

```diff
@@ -1,47 +1,55 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
-PortGroup           cmake 1.0
 PortGroup           github 1.0
 
 github.setup        libsndfile libsndfile 1.0.31
-revision            0
+revision            1
+
 categories          audio
 maintainers         nomaintainer
 platforms           darwin
 
 description         read and write files containing sampled sound
 long_description    \
-	libsndfile is a C library for reading and writing files	\
-	containing sampled sound through one standard library interface. \
-	libsndfile has the following main features: \
-	ability to read and write a large number of file formats, \
-	a simple, elegant and easy to use API, \
-	and on the fly format conversion.
+    libsndfile is a C library for reading and writing files \
+    containing sampled sound through one standard library interface. \
+    libsndfile has the following main features: \
+    ability to read and write a large number of file formats, \
+    a simple, elegant and easy to use API, \
+    and on the fly format conversion.
 license             LGPL-2.1+
+homepage            https://libsndfile.github.io/libsndfile/
+
 checksums           \
-      rmd160  8663673c3f8eaf463c7b499c732c53c9bba03c04 \
-      sha256  f082718e84a27d99c0e349151344b71dc2032e7aa2d799af0da48a8a270aa06a \
-      size    662676
-depends_build-append  port:pkgconfig
-depends_lib           port:flac port:libogg port:libvorbis
+    rmd160  ae3fc5bbcb10a034f3edc1240acacd9f1ec349a7 \
+    sha256  a8cfb1c09ea6e90eff4ca87322d4168cdbe5035cb48717b40bf77e751cc02163 \
+    size    875335
```

Comment:
please have the checksums aligned as the rest of the Portfile

Reply:
If you like, I can re-align the entire file to the same spacing.

Reply:
that's not what I am asking... The majority of the Portfiles have the checksums aligned in the way that `port -dv checksum <portname>` gives them - there is no real eason to start them on a newline.

Reply:
The last dozen or so times I've attempted to use `port checksum` it simply tells me my checksums are wrong, but refuses to tell me what the correct ones are. I've taken to using the openssl tool to calculate them from the correct download manually.

Reply:
that's possible - you do need to add the `d` or `v` flags or perhaps both I don't remember to have it show the correct ones

Reply:
Yeah, I've got that, and the checksums will be indented the same as the rest of the arguments in the file. Hopefully, one indent, 4 spaces, is fine for the keys on the rest of the checksums lines.